var/spack/repos/builtin/packages/gimp/package.py

```diff
@@ -0,0 +1,100 @@
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack.package import *
+
+
+class Gimp(AutotoolsPackage):
+    """GIMP is a cross-platform image editor available for GNU/Linux,
+    macOS, Windows and more operating systems. It is free software,
+    you can change its source code and distribute your changes.
+
+    Whether you are a graphic designer, photographer, illustrator, or
+    scientist, GIMP provides you with sophisticated tools to get your job
+    done. You can further enhance your productivity with GIMP thanks to
+    many customization options and 3rd party plugins."""
+
+    homepage = "https://www.gimp.org"
+    url = "https://download.gimp.org/gimp/v2.10/gimp-2.10.32.tar.bz2"
+
+    maintainers = ["benkirk"]
+
+    conflicts("platform=darwin", msg="spack/GIMP currently requires Linux")
+    conflicts("platform=windows", msg="spack/GIMP currently requires Linux")
+
+    version("2.10.32", sha256="3f15c70554af5dcc1b46e6dc68f3d8f0a6cc9fe56b6d78ac08c0fd859ab89a25")
+    version("2.10.30", sha256="88815daa76ed7d4277eeb353358bafa116cd2fcd2c861d95b95135c1d52b67dc")
+    version("2.10.28", sha256="4f4dc22cff1ab5f026feaa2ab55e05775b3a11e198186b47bdab79cbfa078826")
+    version("2.10.26", sha256="5ddbccf1db462a41df9a26197fcb0d24c7152753a36b3c8b8a9506b4136395f7")
+    version("2.10.24", sha256="bd1bb762368c0dd3175cf05006812dd676949c3707e21f4e6857435cb435989e")
+
+    variant("doc", default=True, description="Build documentation with gtk-doc")
+    variant("ghostscript", default=True, description="Build with ghostscript support")
+    variant("jpegxl", default=True, description="Build with JPEG XL image format support")
+    # variant(
+    #     "libheif",
+    #     default=False,
+    #     description="Build with the libheif HEIF and AVIF file format decoder and encoder."
+    # )
+    variant(
+        "libmng", default=True, description="Build with Multiple-Image Network Graphics support"
+    )
+    variant(
+        "libwmf",
+        default=True,
+        description="Build with libwmf Windows Windows Metafile Format (WMF) support",
+    )
+    variant("libxpm", default=True, description="Build with libxpm support")
+    variant("webp", default=True, description="Build with WebP support")
+    # variant("python",      default=False, description="Build with Python bindings")
+
+    # ref. https://www.gimp.org/source/
+    depends_on("pkgconfig", type="build")
+    depends_on("babl")
+    depends_on("fontconfig@2.12.4:")
+    depends_on("gegl")
+    depends_on("gexiv2")
+    depends_on("ghostscript", when="+ghostscript")
+    depends_on("glib")
+    depends_on("glib-networking")
+    depends_on("gtk-doc", when="+doc")
+    depends_on("gtkplus@2.24.32:2.24.100")
+    depends_on("intltool")
+    depends_on("jpeg")
+    depends_on("libexif")
+    # depends_on("libheif+libde265", when="+libheif")
```

Comment:
I added this package, let me know if you need any help with it

Reply:
@adamjstewart thanks for the offer.  For it to be useful to GIMP, `libheif` needs to be built with both `libde265` and `libx265` decoder / encoder support.

I'm close with the former, but the latter uses some nonstandard paths that I might need som help feeding to Spack properly.  I'll add that to this PR, or follow up with another - what's your preference?

Reply:
I believe all that is required is telling CMake to look in the `./source` directory, rather than the top, to find `CMakeLists.txt`:

```pre
testenv] benkirk@crhtc29(129)$ spack cd x265
[testenv] benkirk@crhtc29(130)$ ls
COPYING  build  doc  readme.rst  source
[testenv] benkirk@crhtc29(131)$ ls source/
CMakeLists.txt  cmake  common  compat  dynamicHDR10  encoder  input  output  profile  test  x265.cpp  x265.def.in  x265.h  x265.pc.in  x265.rc.in  x265_config.h.in  x265cli.h
```

Reply:
Let's do it in a follow-up PR, sounds like it will be tricky